src/controller/CHIPDeviceController.cpp

```diff
@@ -1997,7 +2042,7 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         app::InteractionModelEngine * engine = app::InteractionModelEngine::GetInstance();
         app::ReadPrepareParams readParams(proxy->GetSecureSession().Value());
 
-        app::AttributePathParams readPaths[8];
+        app::AttributePathParams readPaths[9];
```

Comment:
Please add a comment that this array is not allowed to have more than 9 items, since 9 is what the spec has as a minimum requirement on servers.

Reply:
added comment